### PR TITLE
!!! Changes

### PR DESCRIPTION
**Pull request by mhsdesign**

### CODE REVIEW

File: Configuration/Settings.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ CodeQ:
 
     notifications:
       enabled: true
-      service: 'CodeQ\LinkChecker\Service\EmailService'
+      service: CodeQ\LinkChecker\Infrastructure\EmailService
```
Comment (mhsdesign) [inline Configuration/Settings.yaml:18]: my assumption is, that this has not been used and its okay to just change this.

**Comment by mhsdesign** [comment]
email service has not been tested

**Comment by rolandschuetz** [comment]
@mhsdesign One important thing is missing, a crawlnodes should empty the old list of broken links. Otherwise the list just becomes bigger and bigger repeating the same issues

**Comment by mhsdesign** [comment]
Fertig!

**Pull request merged**